modular_skyrat/modules/modular_implants/code/soulcatcher/soulcatcher_component.dm

```diff
@@ -0,0 +1,273 @@
+///Global list containing any and all soulcatchers
+GLOBAL_LIST_EMPTY(soulcatchers)
+
+#define SOULCATCHER_EMOTE_COLOR "#4ba1c9"
+#define SOULCATCHER_SAY_COLOR "#75D5E1"
+#define SOULCATCHER_WARNING_MESSAGE "You have entered a soulcatcher, do not share any information you have received while a ghost. If you have died within the round, you do not know your identity until your body has been scanned, standard blackout policy also applies."
+
+/datum/component/soulcatcher
```

Comment:
This component could have a little bit of documentation explaining what it does.